src/api/action/getActivities.ts

```diff
@@ -141,6 +141,13 @@ export async function getActivitiesAction(
                     return siteHost === replyHost;
                 }
 
+                 // ?filter={type: ['<activityType>:<objectType>:notReply']}
+                 if (filter.criteria && filter.criteria.startsWith('notReply')) {
+                    if (activity.reply_object_url) {
```

Comment:
slightly confusing diff context -`reply_object_url` is not a property an activitypub object, is a property of an activity object returned from `activityMeta.get`